test/unit/runner.spec.js

```diff
@@ -487,6 +487,32 @@ describe('Runner', function() {
         });
         runner.failHook(hook, err);
       });
+
+      it('should not hang if the error message is ridiculously long', function(done) {
```

Comment:
```js
it('should not hang if the error message is ridiculously long single line',
```

Reply:
Well, `err.message` with multiple lines can also cause mocha to hang. What's the point with this wording?

Reply:
Sorry for my misunderstanding, I've pushed specific test case for multilines.